app/javascript/lib/honeycrisp.js

```diff
@@ -103,13 +103,17 @@ var followUpQuestion = (function () {
 
                 // set initial state of follow-ups based on the page
                 $(this).find('input').each(function (index, input) {
-                    if ($(this).attr('data-follow-up') != null) {
-                        if ($(this).is(':checked')) {
-                            $($(this).attr('data-follow-up')).find('input, select').attr('disabled', false);
-                            $($(this).attr('data-follow-up')).show();
+                    var $input = $(this);
```

Comment:
The question-follow-up pattern only assumes that there is one option with the show follow-up option. 
So if two or more radio buttons have data-follow-up="#id-details-fields", the final iteration might be one that is not checked—so it will hide #id-details-fields. This change makes it so that we will load the follow-up if any of the options are checked, not just the last one.

Reply:
@mrotondo and i paired on reviewing this. 

@embarnard, oof good catch on this bug! this is quite hairy code, so thank you for updating it. 

do you mind adding a comment at line 115 just to explain that two questions can refer to the same followup? since this is so tricky, a comment would help. 

for future notice, @mrotondo and i spoke about moving this logic to the update command, but that doesn't need to be included in this PR.

Reply:
definitely! Thank you for reviewing it so thoroughly